org.gnome.Girens.json

```diff
@@ -0,0 +1,297 @@
+{
+    "app-id": "org.gnome.Girens",
```

Comment:
Is Girens really an official GNOME project?

Reply:
No I used or.gnome.Girens, because gnome-builder named my project so. I will change this.

Reply:
You do own g4d.nl?